### PR TITLE
[Irina Zota] Added my first tasks: cicle of area; compare strings; display month b…

### PR DESCRIPTION
**Pull request by irinazota**
Added my first tasks: circle of area; compare strings; display month by number; display number with For.

### CODE REVIEW

File: Irina/src/main/java/FirstTasks/AreaOfCircle.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package FirstTasks;
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/AreaOfCircle.java]: Please have a look on Java Name Convention! Package shuld start with lowercase
https://www.javatpoint.com/java-naming-conventions

File: Irina/src/main/java/FirstTasks/AreaOfCircle.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+/*
+Compute the area of a circle and display it
+ */
+
+public class AreaOfCircle {
+
+
+    public static void main(String[] args) {
+        double Pi = 3.14159;
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/AreaOfCircle.java]: I sugest to use **Math.PI** constant

File: Irina/src/main/java/FirstTasks/AreaOfCircle.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        double radius = 15.35;
+        double circlearea = Pi*(radius*radius);
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/AreaOfCircle.java]: Please name variables acording Java name convention!

File: Irina/src/main/java/FirstTasks/CompareStrings.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+ */
+
+public class CompareStrings {
+    public static void main(String[] args) {
+        String string1 = "Abrandka";
+        String string2 ="Abrandko";
+     if(!(string1.equals(string2))){
+         System.out.println(string1+string2);
+     }
+
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/CompareStrings.java]: Remove empty line

File: Irina/src/main/java/FirstTasks/DisplayMonth.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
+package FirstTasks;
+/*
+Display the month by it number
+ */
+
+public class DisplayMonth {
+    public static void main(String[] args) {
+        int month = 13;
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/DisplayMonth.java]: what is the output of switch in this case?

File: Irina/src/main/java/FirstTasks/DisplayMonth.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                break;
+            case 11:
+                System.out.println("November");
+                break;
+            case 12:
+                System.out.println("December");
+                break;
+        }
+    }
+
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/DisplayMonth.java]: Remove empty line

File: Irina/src/main/java/FirstTasks/DisplayNumberUsingFor.java [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+Display numbers from 1 to 10 using for()
+ */
+
+public class DisplayNumberUsingFor {
+    public static void main(String[] args) {
+        for (int number =1;number<=10;number++){
+            System.out.println(number);
+        }
+    }
+}
```
Comment (venache1) [inline Irina/src/main/java/FirstTasks/DisplayNumberUsingFor.java]: Use Ctrl + Alt + L for code formating. Do this for all classes

Comment (irinazota) [reply Irina/src/main/java/FirstTasks/DisplayNumberUsingFor.java]: Done

File: Irina/src/main/java/FirstTasks/DisplayMonth.java (hunk 6 above)
Comment (irinazota) [reply Irina/src/main/java/FirstTasks/DisplayMonth.java]: Done

File: Irina/src/main/java/FirstTasks/DisplayMonth.java (hunk 5 above)
Comment (irinazota) [reply Irina/src/main/java/FirstTasks/DisplayMonth.java]: Done

File: Irina/src/main/java/FirstTasks/CompareStrings.java (hunk 4 above)
Comment (irinazota) [reply Irina/src/main/java/FirstTasks/CompareStrings.java]: Done

File: Irina/src/main/java/FirstTasks/AreaOfCircle.java (hunk 3 above)
Comment (irinazota) [reply Irina/src/main/java/FirstTasks/AreaOfCircle.java]: Done

File: Irina/src/main/java/FirstTasks/AreaOfCircle.java (hunk 2 above)
Comment (irinazota) [reply Irina/src/main/java/FirstTasks/AreaOfCircle.java]: Done

File: Irina/src/main/java/FirstTasks/AreaOfCircle.java (hunk 1 above)
Comment (irinazota) [reply Irina/src/main/java/FirstTasks/AreaOfCircle.java]: Done

**Review by venache1 — APPROVED** [review]

**Review by Claudia-20 — APPROVED** [review]

File: Irina/src/main/java/FirstTasks/CompareStrings.java [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+/*
+Write a java program to compare two strings lexicographic-ally, ignoring case differences,
+ if strings are different concatenate a given string to the end of another string.
+ */
+
+public class CompareStrings {
+    public static void main(String[] args) {
+        String string1 = "Abrandka";
+        String string2 = "Abrandko";
+        if (!(string1.equals(string2))) {
```
Comment (VitalinaPetrenco) [inline Irina/src/main/java/FirstTasks/CompareStrings.java]: Please cover also "ignoring case differences" as mentioned in this task.

Comment (irinazota) [reply Irina/src/main/java/FirstTasks/CompareStrings.java]: Done

**Review by VitalinaPetrenco — APPROVED** [review]

**Pull request merged**